src/resolve-script-object-to-string.js

```diff
@@ -12,13 +12,22 @@ function resolveScriptObjectToString(script) {
 
 function resolveScriptObjectToScript(script) {
   if (isPlainObject(script)) {
-    if (!isUndefined(script.script)) {
-      return script
-    } else if (!isUndefined(script.default)) {
-      return resolveScriptObjectToScript(script.default)
-    }
+    return resolvePlainObjectToScript(script)
   } else if (isString(script)) {
     return {script}
   }
   return undefined
 }
+
+function resolvePlainObjectToScript(script) {
+  if (!isUndefined(script.hiddenFromHelp)) {
```

Comment:
Huh. I could've sworn that I pushed the truthy-checking version. Obviously I didn't though. :fearful: